### DISCUSSION TITLE
How to add tags to images?

### DISCUSSION THREAD

**Discussion by womzil** [deckerst/aves#125] [Q&A] [answered]
Like in the title, how to do it (if it's possible)? I couldn't find it nor any information how to do so.

**Comment by deckerst** [comment, marked as answer]
It's not possible at the moment.

I intend to work on this soon. You can follow progress on this by subscribing to #1.

**Reply by deckerst** [reply]
It will be possible in the next release to edit tags for the main image formats (JPEG, PNG, GIF, TIFF). Tags can be edited for one item (in the item `Info` page), or in bulk (in the `Collection` page).

**Reply by deckerst** [reply]
Available in Aves v1.5.7.
